draft-ietf-quic-http.md

```diff
@@ -294,11 +294,10 @@ response may contain zero or more header blocks containing the message headers
 of informational (1xx) HTTP responses (see {{!RFC7230}}, Section 3.2 and
 {{!RFC7231}}, Section 6.2).
 
-PUSH_PROMISE frames (see {{frame-push-promise}}) MAY be interleaved with the
-frames of a response message indicating a pushed resource related to the
-response. These PUSH_PROMISE frames are not part of the response, but carry the
-headers of separate HTTP request messages.  See {{server-push}} for more
-details.
+A server MAY interleave one or more PUSH_PROMISE frames (see
+{{frame-push-promise}}) with the frames of a response message indicating a
+pushed resource related to the response. These PUSH_PROMISE frames are not part
```

Comment:
"pushed resource" isn't quite right, we push responses.  You could probably drop from "indicating" onwards and retain the meaning.